mico-core/src/main/resources/application-prod.properties

```diff
@@ -45,3 +45,16 @@ spring.data.neo4j.uri=
 # Redis (will be set by the Kubernetes ConfigMap)
 spring.redis.host=
 spring.redis.port=
+
+# OpenFaaS (will be set by the Kubernetes ConfigMap)
+openfaas.gateway=
+
+# Kafka (will be set by the Kubernetes ConfigMap)
+kafka.bootstrap-servers=
+
+kafka.group-id=mico
+kafka.input-topic=transform-request
```

Comment:
We should as a team agree on an appropriate default input and output topic,

Reply:
You are right. I just took the defaults from the [Kafka-FaaS-Connector](https://github.com/UST-MICO/kafka-faas-connector/blob/master/src/main/resources/application.properties).